cursos/models.py

```diff
@@ -0,0 +1,21 @@
+from django.utils import timezone
+from django.db import models
+
+class Modulos(models.Model):
+    archivos = models.FileField(upload_to="archivos_subidos", max_length=20, null=True)
```

Comment:
Agregar campo de persona asignada (FK to User)

Reply:
Se agrega el campo